### ISSUE TITLE
`telepresence --help` says user daemon is not running

### ISSUE THREAD

**Issue by drewfish** [telepresenceio/telepresence#2735]
**Describe the bug**
I've upgraded to version `2.7.1` (via homebrew). When I run `telepresence --help` it only shows me `telepresence user daemon is not running`. This makes it rather more difficult to see what the top-level commands are.

Oh hmm... `telepresence {command} --help` does work.

**To Reproduce**
1. When I run `telepresence --help`
2. I see "telepresence user daemon is not running"

**Expected behavior**
`telepresence --help` shows a list of top-level commands

**Versions (please complete the following information):**
 - Output of `telepresence version`
   - `v2.7.1 (api v3)`
 - Operating system of workstation running `telepresence` commands
 ```
$ uname -mprsv
Darwin 21.6.0 Darwin Kernel Version 21.6.0: Sat Jun 18 17:07:25 PDT 2022; root:xnu-8020.140.41~1/RELEASE_X86_64 x86_64 i386
```

**Comment by cindymullins-dw** [comment]
Hi @drewfish, this was addressed in 2.7.2.  Could you try upgrading and see if its improved?

**Comment by drewfish** [comment]
Ack... I'm slow to respond 😞 . I'm currently on `2.14.2` and it works well now 👍
